qiskit_experiments/visualization/fit_result_plotters.py

```diff
@@ -21,19 +21,56 @@
 This is just like a function, but allows serialization via Enum.
 """
 
+import dataclasses
 from collections import defaultdict
-from typing import List, Dict, Optional
+from typing import Dict, List, Optional, Tuple
 
-import uncertainties
 import numpy as np
+import uncertainties
 from matplotlib.ticker import FuncFormatter
 from qiskit.utils import detach_prefix
-
-from qiskit_experiments.curve_analysis.curve_data import SeriesDef, FitData, CurveData
+from qiskit_experiments.curve_analysis.curve_data import CurveData, FitData, SeriesDef
 from qiskit_experiments.framework import AnalysisResultData
 from qiskit_experiments.framework.matplotlib import get_non_gui_ax
-from .curves import plot_scatter, plot_errorbar, plot_curve_fit
-from .style import PlotterStyle
+
+from .curves import plot_curve_fit, plot_errorbar, plot_scatter
+
+
+@dataclasses.dataclass
+class PlotterStyle:
+    """A stylesheet specific for :mod:`fit_result_plotters`.
+
+    This style class is used by :mod:`fit_result_plotters`, but not by :class:`BasePlotter` or
```

Comment:
This first sentence seems circular. `PlotterStyle` is for `fit_result_plotters` but it is defined in `fit_result_plotters`. Which part of `fit_result_plotters` is using this? Can you clarify in the docstring?

Reply:
Removed file in f6b6774.